### ISSUE TITLE
Cannot choose between profiles

### ISSUE THREAD

**Issue by sfingali** [AUTOMATIC1111/stable-diffusion-webui-tensorrt#68]
I have generated several engine profiles for SDXL. However, I have no way of selecting the individual engines - all I can choose is one per base model/checkpoint.

I am also unable to delete these engine profiles.

**Comment by IndigoDosSantos** [comment]
> all I can choose is one per base model/checkpoint

Yes, it is somewhat peculiar. I'm not entirely sure if this is a bug. It seems that all engines created for a single checkpoint are grouped together. Based on the txt2img settings, the system automatically selects the best-suited engine from the ones you've created. However, this is just my assumption, and I cannot confirm it. If this is the case, then the "Automatic" selection option in SD Unet appears redundant. What purpose would it serve if the system defaults to automatic selection anyway?

> I am also unable to delete these engine profiles

You can manually remove engines. Navigate to the 'models' folder of your A1111 and find the 'Unet-onnx' and 'Unet-trt' directories. From there, you can delete the engines you no longer require.

**Comment by sfingali** [comment]
Thank you, I have manually deleted them, there wasn't documentation as to location but found on a YouTube video before your comment. All working with speed increase. If we get get multi-gpu working with A1111 we'll be cooking.
